product_sold_by_delivery_week/models/sale_order.py

```diff
@@ -40,5 +40,5 @@ def _compute_weekly_sold_delivered_shown(self):
         for line in to_process_lines:
             partner = line.get_partner_for_reporting()
             line.weekly_sold_delivered_shown = _format_weekly_string(
-                partner_products_weekly[partner][line.product_id]
+                partner_products_weekly[partner].get(line.product_id, False)
```

Comment:
Does it need to be  `False` or could it be `None`?

Reply:
Changed